packages/notifier/src/publish-kit.js

```diff
@@ -298,6 +304,13 @@ const advanceDurablePublishKit = (context, value, targetStatus = 'live') => {
   }
   const { tailP: currentP, tailR: resolveCurrent } =
     provideDurablePublishKitEphemeralData(state, facets);
+  const commit = resolution => {
+    // Invoke a direct callback, but ignore the result.
+    if (onUpdate) {
+      void callE(onUpdate, Promise.resolve(resolution)).then(sink, sink);
```

Comment:
I'd prefer not sinking rejections here.  A `SES_UNHANDLED_REJECTION` is more helpful for diagnosing errors in the callback.
```suggestion
      void callE(onUpdate, Promise.resolve(resolution));
```